lib/scala/pkg/src/main/scala/org/enso/pkg/Config.scala

```diff
@@ -76,22 +79,23 @@ object Contact {
 
 /** Represents a package configuration stored in the `package.yaml` file.
   *
-  * @param name package name
-  * @param version package version
-  * @param ensoVersion version of the Enso engine associated with the package,
-  *                    can be set to `default` which defaults to the locally
-  *                    installed version
-  * @param license package license
-  * @param authors name and contact information of the package author(s)
-  * @param maintainers name and contact information of current package
-  *                   maintainer(s)
+  * @param name         package name
+  * @param version      package version
+  * @param ensoVersion  version of the Enso engine associated with the package,
+  *                     can be set to `default` which defaults to the locally
+  *                     installed version
+  * @param license      package license
+  * @param authors      name and contact information of the package author(s)
+  * @param maintainers  name and contact information of current package
+  *                     maintainer(s)
   * @param dependencies a list of package dependencies
   * @param originalJson a Json object holding the original values that this
   *                     Config was created from, used to preserve configuration
   *                     keys that are not known
   */
 case class Config(
   name: String,
+  namespace: String,
```

Comment:
Namespace is not documented.

I think it's important to note here that this is just a temporary workaround that will be removed once it is possible.
Also that it falls back to `local` if not provided.